test/bin/build_images.sh

```diff
@@ -31,17 +31,27 @@ osbuild_logs() {
 }
 
 extract_container_images() {
-    local -r version=$1
-    local -r repo_url=$2
-    local -r outfile=$3
+    local -r version=$1    # full version
+    local -r repo_spec=$2  # repo name, path, or URL
+    local -r outfile=$3    # destination file
 
     echo "Extracting images from ${version}"
     mkdir -p "${IMAGEDIR}/release-info-rpms"
     pushd "${IMAGEDIR}/release-info-rpms"
     dnf_options=""
-    if [[ -n ${repo_url} ]]; then
-        local -r repo_name="$(basename "${repo_url}")"
-        dnf_options="--repofrompath ${repo_name},${repo_url} --repo ${repo_name}"
+    local -r repo_name="$(basename "${repo_spec}")"
+    if [[ "${repo_spec}" =~ ^https://.* ]]; then
+        # If the spec is a URL, set up the arguments to point to that location.
+        dnf_options="--repofrompath ${repo_name},${repo_spec} --repo ${repo_name}"
+    elif [[ "${repo_spec}" =~ ^/.* ]]; then
+        # If the spec is a path, set up the arguments to point to that path.
+        dnf_options="--repofrompath ${repo_name},${repo_spec} --repo ${repo_name}"
+    elif [[ -n ${repo_spec} ]]; then
```

Comment:
```suggestion
    elif [[ -n "${repo_spec}" ]]; then
```

Reply:
Do we expect this value to be empty? 
If not, we should probably add code that reports an error on an empty value, i.e. the last `else` in the block.

Reply:
It's OK to have an empty value. In that case the query isn't limited to a specific repository.